src/syft/lib/python/string.py

```diff
@@ -155,9 +156,11 @@ def islower(self) -> PyPrimitive:
 
     @syft_decorator(typechecking=True, prohibit_args=True)
     def isascii(self) -> PyPrimitive:
-        # Got mypy issues saying that isascii is not defined in superclass
-        res = self.data.isascii()
-        return PrimitiveFactory.generate_primitive(value=res)
+        if sys.version_info >= (3, 7):
+            res = self.data.isascii()
```

Comment:
Q: Could you try here: ```super().isascii()```

Reply:
Yup, fixing it now!

Reply:
Weird...
```
src/syft/lib/python/string.py:160: error: "isascii" undefined in superclass
Found 1 error in 1 file (checked 194 source files)
```

There was a comment here about this before I think.

Reply:
Im going to add # type: ignore since this seems silly.

Reply:
Yes - I added the comment because I do not know how that ```isascii``` is working :\